components/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/SAMLSSOUIConstants.java

```diff
@@ -63,6 +63,11 @@
     public static final String SESSION_ATTRIBUTE_NAME_APPLICATION_CERTIFICATE = "applicationCertificate";
     public static final String ENABLE_SAML2_ECP = "enableSAML2ECP";
 
+    public static final String SLO_TYPE = "singleLogoutType";
+    //Front Channel Logout Methods
+//    public static final String HTTP_REDIRECT_BINDING = "HTTPRedirectBinding";
```

Comment:
shall we remove commented out code if not required?

Reply:
This is fixed in the master - https://github.com/wso2-extensions/identity-inbound-auth-saml/blob/master/components/org.wso2.carbon.identity.sso.saml.ui/src/main/java/org/wso2/carbon/identity/sso/saml/ui/SAMLSSOUIConstants.java#L69